material-overrides/assets/stylesheets/moonbeam.css

```diff
@@ -1118,23 +1118,34 @@ html .md-typeset details.function div.tabbed-set,
 }
 
 /* Feedback widget styling */
-.md-feedback {
-  margin: 1em auto 2em;
-  text-align: center;
-  border: var(--border-width) solid var(--border-purple-bright);
+.feedback-actions-container {
+  margin-top: 2rem;
+  width: calc(100% - 2em);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1.5em;
+  box-sizing: border-box;
+  border: 1px solid var(--purple-dark-tint-20);
   border-radius: var(--border-radius);
-  padding: 1em 4em;
-  width: fit-content;
+}
+
+.md-feedback {
+  margin-top: 0;
+  margin-bottom: 0;
 }
 
 .md-feedback__title {
+  margin-bottom: 0.75em;
   font-family: var(--md-text-font);
-  font-size: 80%;
+  font-size: 12px;
   font-weight: 400;
 }
 
 .md-feedback__list {
-  justify-content: space-between;
+  margin-top: 0.75em;
+  gap: 1em;
+  margin-bottom: 0.75em;
 }
 
 .md-feedback__icon {
```

Comment:
There are some stylings here that are no longer relevant because the True/False labels were deleted. Can you please take a look?

Reply:
Also there is some padding that was applied to the thumbs up and thumbs down icons that are making it so the icons aren't centered in their border.

Reply:
Once you fix the padding, please make sure that the icons are the same size. I'd scale them down a little so they're not so big